UM/Trust.py

```diff
@@ -285,6 +285,16 @@ def removeCached(path: str) -> bool:
             Logger.logException("e", "Removal of pycache for unbundled path '{0}' failed.".format(path))
         return False
 
+    @staticmethod
+    def isPathInLocation(location: str, path: str) -> bool:
+        try:
+            canonical_location = os.path.normpath(location)
```

Comment:
Since we're now at python 3.8, pathlib has much better support. Perhaps we should consider using that more (it at the very least has a nicer interface / convenience functions!). It's probably a bit out of scope for this issue though.

Reply:
I feel your pain and yes out-of-scope for this ticket. But nice to keep in mind for future reference.